spec/cassettes/ActiveCampaign_API_Fields/_update_field/returns_a_user_hash.yml

```diff
@@ -0,0 +1,210 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: "<API_ENDPOINT>/fields"
+    body:
+      encoding: UTF-8
+      string: '{"field":{"title":"Custom text - updated","type":"text","descript":"A
+        custom text field - updated","isrequired":"1","perstag":"perstag","defval":"Default
+        value - updated","visible":"1","ordernum":"3"}}'
+    headers:
+      User-Agent:
+      - ActiveCampaign Ruby Client (v3.0.0)
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      Api-Token:
+      - "<API_KEY>"
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 201
+      message: Created
+    headers:
+      Date:
+      - Mon, 25 May 2020 08:22:11 GMT
+      Content-Type:
+      - application/json
+      Content-Length:
+      - '462'
+      Connection:
+      - keep-alive
+      Set-Cookie:
+      - PHPSESSID=a2e1b4aaacc1d0df5237ca1a8a4d8728; path=/; HttpOnly
+      - __cfduid=dfcdb2f71579a428692bb94ec426175341590394931; expires=Wed, 24-Jun-20
+        08:22:11 GMT; path=/; domain=.api-us1.com; HttpOnly; SameSite=Lax
+      - em_acp_globalauth_cookie=7ab2bf53-db21-4ed6-8417-f5c11476cd5d; path=/; domain=.camplify1589433994.api-us1.com;
+        secure; httponly
+      - em_acp_globalauth_cookie=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0;
+        path=/; domain=.camplify1589433994.api-us1.com
+      - em_acp_globalauth_cookie=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0;
+        path=/; domain=.camplify1589433994.api-us1.com
+      Cache-Control:
+      - no-store, no-cache, must-revalidate
+      Expires:
+      - Thu, 19 Nov 1981 08:52:00 GMT
+      Pragma:
+      - no-cache
+      X-Envoy-Upstream-Service-Time:
+      - '269'
+      X-Request-Id:
+      - a17a1551-895e-4dbc-93ba-6bc1c8f93aa7
+      Cf-Cache-Status:
+      - DYNAMIC
+      Expect-Ct:
+      - max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
+      Server:
+      - cloudflare
+      Cf-Ray:
+      - 598dd8e1c90265eb-SYD
+      Cf-Request-Id:
+      - 02ec85e11b000065eb0d050200000001
+    body:
+      encoding: UTF-8
+      string: '{"field":{"type":"text","title":"Custom text - updated","descript":"A
+        custom text field - updated","isrequired":"1","perstag":"PERSTAG","defval":"Default
+        value - updated","visible":"1","ordernum":1,"cdate":"2020-05-25T03:22:11-05:00","udate":"2020-05-25T03:22:11-05:00","links":{"options":"https:\/\/camplify1589433994.api-us1.com\/api\/3\/fields\/213\/options","relations":"https:\/\/camplify1589433994.api-us1.com\/api\/3\/fields\/213\/relations"},"id":"213"}}'
+    http_version: null
+  recorded_at: Mon, 25 May 2020 08:22:11 GMT
+- request:
+    method: put
+    uri: "<API_ENDPOINT>/fields/213"
+    body:
+      encoding: UTF-8
+      string: '{"field":{"title":"Custom text - updated","type":"text","descript":"A
```

Comment:
My previous comment is for the create. All interactions for one test (`it "returns a field hash"`) are recorded into the same cassette. If you compare this request to the create they are pretty much identical except for the id in the URL and the PUT instead of POST.

What I learned with ActiveCampaign is that their API leaves a whole lot to be wished for still and making sure that we test the specific request properly is of the utmost importance to avoid support requests down the line :)

Reply:
> All interactions for one test (it "returns a field hash") are recorded into the same cassette.

Ahh, I understand now. Thanks very much for the explanation!